libnmstate/validator.py

```diff
@@ -109,6 +113,15 @@ def validate_link_aggregation_state(desired_state, current_state):
                 specified_slaves |= slaves
 
 
+def validate_interface_unique_name(state):
```

Comment:
This should be `validate_unique_interface_name`.

Reply:
Done